docs/command_line_reference.rst

```diff
@@ -509,3 +510,67 @@ When you run ``esrally list races``, this will show up again::
     20160518T112341Z  pmc                         append-no-conflicts  defaults  disk:SSD,data_node_count:4
 
 This will help you recognize a specific race when running ``esrally compare``.
+
+.. _command_line_reference_advanced_topics:
+
+Advanced topics
+---------------
+
+``target-hosts``
+~~~~~~~~~~~~~~~~
+
+Rally can also create client connections for multiple Elasticsearch clusters.
+This is only useful if you want to create :ref:`custom runners <adding_tracks_custom_runners>` that execute operations against remote clusters, for example for Cross Cluster Search or Cross Cluster Replication.
+
+To define the host:port pairs for additional clusters with ``target-hosts`` you can specify either a JSON filename (ending in ``.json``) or an inline JSON string. The JSON object should be a collection of name:value pairs. ``name`` is string for the cluster name and there **must be** one ``default``.
+
+Examples:
+
+* json file: ``--target-hosts="target_hosts1.json"``::
+
+    { "default": ["127.0.0.1:9200","10.127.0.3:19200"] }
+
+* json file defining two clusters: ``--target-hosts="target_hosts2.json"``::
+
+    {
+      "default": [
+        {"host": "127.0.0.1", "port": 9200},
+        {"host": "127.0.0.1", "port": 19200}
+      ],
+      "remote":[
+        {"host": "10.127.0.3", "port": 9200},
+        {"host": "10.127.0.8", "port": 9201}
+      ]
+    }
+
+* json inline string defining two clusters::
+
+    --target-hosts="{\"default\":[\"127.0.0.1:9200\"],\"remote\":[\"127.0.0.1:19200\",\"127.0.0.1:19201\"]}"
+
+.. NOTE::
+   **All** internal :ref:`track operations <track_operations>` will use the connection to the ``default`` cluster. However, you can utilize the client connections to the additional clusters in your :ref:`custom runners <adding_tracks_custom_runners>`.
```

Comment:
Nit: It took me a while to understand the term "internal track operations". Maybe use "built-in operations" instead?

Reply:
:+1: